extension.js

```diff
@@ -182,30 +200,27 @@ function getProxyConfig() {
  */
 function activate(context) {
   let translateText = vscode.commands.registerCommand(
-    "extension.translateText",
+    "vscodeGoogleTranslate.translateText",
```

Comment:
it will break the already configured settings for current users

Reply:
Hm I think the only breaking change here would be if someone already configured their own hotkeys for each command. However, another possible breaking change is if they install another extension with a registered command "extension.translateText". At least that what I was trying to prevent. I'll change back

Reply:
I think you’re right but we maybe need to write a release note that will show in vscode on update to display possible breaking changes

Reply:
I think leaving the names as "extension.foobarbaz" is okay. I think that enough people will have configured their own hotkeys that this would legitimately break enough people that it isn't worth it